features/search-replace-export.feature

```diff
@@ -26,6 +26,20 @@ Feature: Search / replace with file export
       INSERT INTO `wp_options`
       """
 
+    When I run `wp search-replace example.com example.net --skip-tables=wp_opt\?ons,wp_post\* --export`
+    Then STDOUT should not contain:
```

Comment:
The tests should contain a positive as well, like `Then STDOUT should contain: """wp_users"""`.
Otherwise, the above test would even succeed if everything was filtered out or the output was completely missing.